roles/kubernetes/node/tasks/main.yml

```diff
@@ -134,6 +134,14 @@
   tags:
     - kube-proxy
 
+- include_tasks: "{{ cloud_provider }}-credential-check.yml"
```

Comment:
Is `cloud_provider` changing during runtime of the playbook?

Else use import_tasks which is evaluted upon playbook start and much faster

Reply:
yes, `cloud_provider` is static